tests/test_mapping.py

```diff
@@ -0,0 +1,27 @@
+#%% Import
+
+import numpy as np
+from numpy.testing import assert_almost_equal, assert_array_equal
+from pytest_cases import parametrize_with_cases
+
+from imod_coupler.utils import create_mapping
+
+
+#%% Test
+@parametrize_with_cases(
+    "src_idx,tgt_idx,nsrc,ntgt,operator,expected_map_dense,expected_mask"
+)
+def test_create_mapping(
+    src_idx, tgt_idx, nsrc, ntgt, operator, expected_map_dense, expected_mask
```

Comment:
Many of the names here are not very obvious

Reply:
The argument names are equivalent to the `create_mapping` function, I'll write some docstring pointing to the `create_mapping` function.

Reply:
One more reason to refactor that function :P